<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_rpt_report_servantpromotquery_Dialog</class>
 <widget class="QDialog" name="w_rpt_report_servantpromotquery_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,0,0">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>开始时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="lds_ComboBox" name="comboBox_start">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>结束时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="lds_ComboBox" name="comboBox_end">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color:purple;</string>
        </property>
        <property name="text">
         <string>服务员</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_ComboBox" name="comboBox_operator">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar" name="widget"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string/>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="lds_tableview_sum" name="tableView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string/>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="lds_tableview_sum" name="tableView_2"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>制表人</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_operator">
          <property name="text">
           <string>制表人</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color:purple;</string>
          </property>
          <property name="text">
           <string>制表时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_operattime">
          <property name="text">
           <string>制表时间</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_tableview_sum</class>
   <extends>QTableView</extends>
   <header>lds_tableview_sum.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_operator</tabstop>
  <tabstop>comboBox_start</tabstop>
  <tabstop>comboBox_end</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>tableView_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
